src/payment/strategies/amazon-payv2/amazon-payv2-payment-processor.ts

```diff
@@ -21,6 +21,23 @@ export default class AmazonPayv2PaymentProcessor {
         return this._configureWallet();
     }
 
+    deinitialize(): Promise<void> {
+        this._amazonPayv2SDK = undefined;
+
+        return Promise.resolve();
+    }
+
+    bindButton(buttonId: string, sessionId: string): void {
+        if (!this._amazonPayv2SDK) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        this._amazonPayv2SDK.Pay.bindChangeAction(buttonId, {
+            amazonCheckoutSessionId: sessionId,
+            changeAction: 'changeAddress',
```

Comment:
Why are you hard coding the action to be `'changeAddress'`. I assume it is one of the actions that you can perform. I think it is better for this to be passed in as a parameter instead so that it is immediately clear to the caller that calling this method will bind the "changeAddress" action to the specified button.